client/asset/interface.go

```diff
@@ -16,19 +16,18 @@ import (
 type WalletTrait uint64
 
 const (
-	WalletTraitRescanner           WalletTrait = 1 << iota // The Wallet is an asset.Rescanner.
-	WalletTraitNewAddresser                                // The Wallet can generate new addresses on demand with NewAddress.
-	WalletTraitLogFiler                                    // The Wallet allows for downloading of a log file.
-	WalletTraitFeeRater                                    // Wallet can provide a fee rate for non-critical transactions
-	WalletTraitAccelerator                                 // This wallet can accelerate transactions using the CPFP technique
-	WalletTraitRecoverer                                   // The wallet is an asset.Recoverer.
-	WalletTraitWithdrawer                                  // The Wallet can withdraw a specific amount from an exchange wallet.
-	WalletTraitSweeper                                     // The Wallet can sweep all the funds, leaving no change.
-	WalletTraitRestorer                                    // The wallet is an asset.WalletRestorer
-	WalletTraitRedemptionConfirmer                         // The wallet has a process to confirm a redemption.
```

Comment:
These do not need to be kept in order, correct?

Reply:
They do not afaik, but I mean check and adjust the frontend where they are mirrored.

Reply:
Ok sorry, missed that. Will do.